chart/kubeapps/templates/NOTES.txt

```diff
@@ -8,6 +8,21 @@ Kubeapps can be accessed via port {{ .Values.frontend.service.port }} on the fol
 
    {{ template "kubeapps.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
 
+{{- $orphanSecrets := false -}}
+{{- range .Values.apprepository.initialRepos }}
+{{- if or .caCert .authorizationHeader }}
+{{- $orphanSecrets = true -}}
```

Comment:
this gets rejected by the template-test but I am not sure why. This is used to print the caveat only if there is a `.caCert` or `.authorizationHeader` set.

Reply:
So this works with `helm install` but not with `helm template`?

Reply:
I don't think we should require 2.12.0 just for some documentation, I would try to dig more into Go templates here to find another solution.

Reply:
I have been checking the documentation and I didn't find anything useful. The only thing I think  we can do is to print a line per app repository that contains a secret.